packages/lisk-transactions/test/4_register_multisignature_account.js

```diff
@@ -312,6 +312,111 @@ describe('#registerMultisignatureAccount transaction', () => {
 				return Promise.resolve();
 			});
 
+			describe('shoud throw error', () => {
+				describe('when #lifetime', () => {
```

Comment:
`#` is used fairly inconsistently, but generally indicates a function or method under test. This should just be `lifetime`.